support-frontend/assets/components/ticker/contributionTicker.jsx

```diff
@@ -128,7 +128,8 @@ export default class ContributionTicker extends Component<PropTypes, StateTypes>
   increaseTextCounter = () => {
     const nextCount = this.state.count + Math.floor(this.state.count / 100);
 
-    if (this.dataFromServer.totalContributed && nextCount >= this.dataFromServer.totalContributed) {
+    if (this.dataFromServer.totalContributed &&
```

Comment:
it's hard to tell just from the code what that `nextCount <= this.state.count` is actually doing - is there any way that variable names/comments could make it clear?

Reply:
I've just pushed a commit that attempts to make it clearer

Reply:
thanks!